lib/connections/structs.go

```diff
@@ -193,9 +194,11 @@ type genericListener interface {
 type Model interface {
 	protocol.Model
 	AddConnection(conn protocol.Connection, hello protocol.Hello)
+	NumConnections() int
 	Connection(remoteID protocol.DeviceID) (protocol.Connection, bool)
 	OnHello(protocol.DeviceID, net.Addr, protocol.Hello) error
 	GetHello(protocol.DeviceID) protocol.HelloIntf
+	DeviceStatistics() (map[string]stats.DeviceStatistics, error)
```

Comment:
Now that it's added to the public interface, maybe it's the time to change the key from `string` to `protocol.DeviceID`?